### ISSUE TITLE
SyntaxError: Invalid regular expression

### ISSUE THREAD

**Issue by jeongsd** [junedomingo/react-native-rename#63]
#### What happened
```sh
$ react-native init test
$ cd test
$ react-native-rename "Travel App"
Something went wrong:  SyntaxError: Invalid regular expression: /^([\p{Letter}\p{Number}])+([\p{Letter}\p{Number}\s]+)$/: Invalid escape
    at new RegExp (<anonymous>)
    at Command.<anonymous> (/Users/work/.nvm/versions/node/v8.11.1/lib/node_modules/react-native-rename/lib/index.js:120:19)
    at Command.listener (/Users/work/.nvm/versions/node/v8.11.1/lib/node_modules/react-native-rename/node_modules/commander/index.js:315:8)
    at emitOne (events.js:116:13)
    at Command.emit (events.js:211:7)
    at Command.parseArgs (/Users/work/.nvm/versions/node/v8.11.1/lib/node_modules/react-native-rename/node_modules/commander/index.js:656:12)
    at Command.parse (/Users/work/.nvm/versions/node/v8.11.1/lib/node_modules/react-native-rename/node_modules/commander/index.js:474:21)
    at /Users/work/.nvm/versions/node/v8.11.1/lib/node_modules/react-native-rename/lib/index.js:315:6
    at <anonymous>
```

#### react-native info
```sh
  React Native Environment Info:
    System:
      OS: macOS 10.14
      CPU: x64 Intel(R) Core(TM) i5-7600 CPU @ 3.50GHz
      Memory: 686.64 MB / 16.00 GB
      Shell: 5.3 - /bin/zsh
    Binaries:
      Node: 8.11.1 - ~/.nvm/versions/node/v8.11.1/bin/node
      Yarn: 1.10.1 - /usr/local/bin/yarn
      npm: 5.6.0 - ~/.nvm/versions/node/v8.11.1/bin/npm
      Watchman: 4.9.0 - /usr/local/bin/watchman
    SDKs:
      iOS SDK:
        Platforms: iOS 12.1, macOS 10.14, tvOS 12.1, watchOS 5.1
    IDEs:
      Android Studio: 3.2 AI-181.5540.7.32.5056338
      Xcode: 10.1/10B61 - /usr/bin/xcodebuild
    npmPackages:
      react: 16.6.0-alpha.8af6728 => 16.6.0-alpha.8af6728
      react-native: 0.57.4 => 0.57.4
    npmGlobalPackages:
      create-react-native-app: 2.0.2
      react-native-cli: 2.0.1
      react-native-rename: 2.3.1
      svg-to-react-native-cli: 0.0.3
```

**Comment by 91ayush** [comment]
Same here

**Comment by NikhilSaini38** [comment]
Same problem

**Comment by jmnsf** [comment]
Same here, looks like a regression from https://github.com/junedomingo/react-native-rename/commit/fb0445d5f627373756d72d4ac6972db8f5f4fc1b

**Comment by zainulhasan** [comment]
Same problem

**Comment by tushgup** [comment]
Same error.

**Comment by rendy14s** [comment]
same here

**Comment by traxx10** [comment]
Same here

**Comment by efleurine** [comment]
The problem might be related to @https://github.com/slevithan/xregexp/issues/197

**Comment by luisfuertes** [comment]
Any update?

**Comment by rendy14s** [comment]
@luisfuertes nope

**Comment by azizimusa** [comment]
npm uninstall react-native-rename -g
npm install react-native-rename@2.2.2 -g

**Comment by ajcrites** [comment]
Node 10 and 11 (LTS and current versions) support Unicode regular expressions. If you want to use the latest versions, you can run it using Node 10 or 11. Otherwise, install an earlier version (e.g. 2.2.2).

I'll see if I can publish a fix for this, though.

**Comment by ajcrites** [comment]
Fixed in v2.3.2

**Comment by rendy14s** [comment]
Thank you 👍
